src/main/java/org/apache/cassandra/sidecar/MainModule.java

```diff
@@ -194,6 +197,19 @@ public Router vertxRouter(Vertx vertx,
         router.get(API_V1_VERSION + listSnapshotFilesRoute)
               .handler(listSnapshotFilesHandler);
 
+        final String keyspacesRoute = "/keyspaces";
+        router.get(API_V1_VERSION + keyspacesRoute)
+              .handler(keyspacesHandler);
+
+        // handles:
+        // - /api/v1/keyspace/:keyspace and
+        // - /api/v1/keyspace/:keyspace/table/:table
+        router.route()
+              .method(HttpMethod.HEAD)
+              .method(HttpMethod.GET)
+              .pathRegex("\\/api\\/v1\\/keyspace\\/(?<keyspace>[^\\/]+)(\\/table\\/(?<table>[^\\/]+))?")
```

Comment:
I'm not too happy about having the regex. Maybe we should just use two strings instead:
```suggestion
              .path(API_V1_VERSION + "/keyspace/:keyspace");

        router.route()
              .method(HttpMethod.HEAD)
              .method(HttpMethod.GET)
              .path(API_V1_VERSION + "/keyspace/:keyspace/table/:table");
```

Reply:
+1 to the brain relief.